src/target/stm32l4.c

```diff
@@ -496,12 +497,27 @@ static const uint32_t stm32wl_default_options_values[7] = {
 	0xffffff00U,
 };
 
+static const uint32_t stm32wb_default_options_values[10] = {
```

Comment:
Apologies for the extra review note, we missed this yesterday but `lint` does not - please make all hexadecimal constants lower-case and suffix them with `U` so they are unsigned and don't undergo signed-unsigned conversion.

Please then squash the fixup commits into the original commit.

Reply:
allright, no worries overlooked it myself as well, will update it in a bit.